src/System.Runtime.Serialization.Xml/tests/System.Runtime.Serialization.Xml.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{30CAB353-089E-4294-B23B-F2DD1D945654}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
Also, if you do the above request, can you make sure that all of those projects have a uapaot configuration? If they don't then this Constant will never be defined as TargetGroup will never be uapaot.